lms/envs/common.py

```diff
@@ -362,7 +362,7 @@
     'ENABLE_READING_FROM_MULTIPLE_HISTORY_TABLES': True,
 
     # WIP -- will be removed in Ticket #TNL-4750.
-    'ENABLE_TIME_ZONE_PREFERENCE': False,
+    'ENABLE_TIME_ZONE_PREFERENCE': True,
```

Comment:
FYI - you can enable this in your own `private.py` in `lms/envs/` folder.  The `private.py` file is ignored by git since it's listed in our `.gitignore` file.  This way, you don't have to have a temporary commit and then remember to revert the change.